RecoMET/METProducers/src/HcalNoiseInfoProducer.cc

```diff
@@ -112,6 +113,49 @@ HcalNoiseInfoProducer::HcalNoiseInfoProducer(const edm::ParameterSet& iConfig) :
     edm::LogWarning("HCalNoiseInfoProducer") << " forcing fillRecHits to be true if fillDigis is true.\n";
   }
 
+  // get the fiber configuration vectors
+  std::vector<int> TmpLaserMonDetTypeList = iConfig.getParameter<std::vector<int> >("LaserMonDetTypeList");
```

Comment:
Local variables should start with lowercase letters, as well as the config parameters